src/Backends/Helmholtz/VLERoutines.cpp

```diff
@@ -1173,7 +1173,16 @@ void SaturationSolvers::successive_substitution(HelmholtzEOSMixtureBackend &HEOS
             df += dfdK*(deriv_liq-deriv_vap);
         }
         
-        change = -f/df;
+        if (std::abs(df) <= 1e-12) { // To avoid dividing by 0
+            change = -f;
```

Comment:
I was not fully sure about what to do with this case.
I don't have the understanding on the physics involved in these variables, so I don't really know which value they are expected to have.

So if f is in tolerance, we finish the loop (the the K that have just been computed is used), and if not, we error out.

I don't think I ever experienced the case where df would be small without f being converged.
So I'll change to error out when it happens with an nicer message (`very` with only one `r`and value of  `f` and `df` displayed).
That way, if it happens, we will have the informations to know what to do, and if it doesn't happen, the problem is solved. (case where df where too small where already quite rare).